crates/pallets/starknet/src/lib.rs

```diff
@@ -715,6 +747,17 @@ pub mod pallet {
             // otherwise we have a nonce error and everything fails.
             // Once we have a real fee market this is where we'll chose the most profitable transaction.
 
+            // Firstly let's check that we call the right function.
+            if let Call::submit_gas_price_unsigned { gas_price: new_gas_price } = call {
+                // We can only accept transactions with a priority lower than `UnsignedPriority`.
+                return ValidTransaction::with_tag_prefix("starknet")
+                    .priority(u64::MAX)
+                    .and_provides(new_gas_price)
```

Comment:
and_provies adds a tag to the transaction don't set that

Reply:
OKay